include/silo/storage/column/pango_lineage_column.h

```diff
@@ -15,11 +15,11 @@ class PangoLineageColumn : public Column {
   public:
    template <class Archive>
    [[maybe_unused]] void serialize(Archive& archive, const unsigned int /* version */) {
-      archive & values;
-      archive & value_id_lookup;
-      archive & id_value_lookup;
-      archive & indexed_values;
-      archive & indexed_sublineage_values;
+      archive& values;
```

Comment:
You could simply squash this commit into the previous one -> less noise in the Git history.